lib/add-tests.coffee

```diff
@@ -52,6 +52,49 @@ addTests = (raml, tests, hooks, parent, callback, testFactory) ->
     # In case of issue #8, resource does not define methods
     resource.methods ?= []
 
+    if sortFirst && resource.methods.length > 1
```

Comment:
I'm thinking in [SOLID](https://en.wikipedia.org/wiki/SOLID_(object-oriented_design)) paradigm. For me, ordering methods is not responsibility of add-test file. I think, we can make parser.coffee, which will be a wrapper on the raml-parser and have this functionality. In this case, the code will be more readable. What do you think?